File: Mathmatical/fabonic.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cin>>n;
    
    if(n == 1) {
        cout<<1;
        return 0;
    }
    
    if(n == 2) {
        cout<<1 <<" "<< 1;
        return 0;
    }
    
    int a = 1, b = 1, temp;
    cout << 1 << " " << 1 << " ";
    
    for(int i = 2; i < n; i++){
        temp = a + b;
        a = b;
        b = temp;
        cout << temp << " ";
    }
    
    return 0;
}

File: Mathmatical/fabonic.py
n=int(input("enter n for fibonic\n"))

a=1
b=1
if n==1:
    print(1)
elif n==2:
    print(str(1) + " " + str(1))
else:
    print(f"1 1",end=' ')
    for _ in range(n-2):
        temp=a+b
        print(f"{temp}",end=' ')
        a=b
        b=temp


